.github/workflows/cla-assistant.yaml

```
@@ -0,0 +1,47 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: read
+  contents: read # this can be 'read' if the signatures are in remote repository
+  pull-requests: write
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - id: fetch-token
+        name: Fetch installation token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CLA_ASSISTANT_APP_ID }}
+          private-key: ${{ secrets.CLA_ASSISTANT_APP_PRIVATE_KEY }}
+          repositories: cla-signatures
+      - id: get-membership
+        name: "Determine gravitational membership"
+        env:
+          GITHUB_TOKEN: ${{ steps.fetch-token.outputs.token }}
+          USER: ${{ github.actor }}
+        continue-on-error: true
+        # Use the "Check Membership" API for user
+        # If the user is a member of the org expect a 204 HTTP Status Code
+        # If the above is success the process will exit 0
+        run: |
+          gh api "orgs/gravitational/members/${USER}"
```

```suggestion
          gh api "orgs/gravitational/members/${USER}" > /dev/null
```

This doesn't contain much sensitive information currently, but would probably be a good idea to either capture the output or trash it in case something changes in the future